### PR TITLE
Install boto3 from apk testing

### PR DESCRIPTION
**Pull request by mtb-xt**
@tboerger I've screwed up my previous PR, sorry. Here are the changes.

### CODE REVIEW

File: docker/Dockerfile.linux.amd64 [hunk 1]
```diff
@@ -5,7 +5,7 @@ LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
   org.label-schema.vendor="Drone.IO Community" \
   org.label-schema.schema-version="1.0"
 
-RUN apk add --no-cache bash git curl rsync openssh-client py3-pip py3-requests python3-dev libffi-dev libressl-dev libressl build-base && \
+RUN apk add --no-cache bash git curl rsync openssh-client py3-pip py3-requests py3-boto3@testing python3-dev libffi-dev libressl-dev libressl build-base && \
   pip3 install -U pip && \
   pip3 install ansible==2.8.4 && \
   apk del python3-dev libffi-dev libressl-dev build-base
```
Comment (tboerger) [inline docker/Dockerfile.linux.amd64]:
```patch
-  apk del python3-dev libffi-dev libressl-dev build-base
+  apk del --no-cache python3-dev libffi-dev libressl-dev build-base
```

File: docker/Dockerfile.linux.arm [hunk 2]
```diff
@@ -5,7 +5,7 @@ LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
   org.label-schema.vendor="Drone.IO Community" \
   org.label-schema.schema-version="1.0"
 
-RUN apk add --no-cache bash git curl rsync openssh-client py3-pip py3-requests python3-dev libffi-dev libressl-dev libressl build-base && \
+RUN apk add --no-cache bash git curl rsync openssh-client py3-pip py3-requests py3-boto3@testing python3-dev libffi-dev libressl-dev libressl build-base && \
   pip3 install -U pip && \
   pip3 install ansible==2.8.4 && \
   apk del python3-dev libffi-dev libressl-dev build-base
```
Comment (tboerger) [inline docker/Dockerfile.linux.arm]:
```patch
-  apk del python3-dev libffi-dev libressl-dev build-base
+  apk del --no-cache python3-dev libffi-dev libressl-dev build-base
```

File: docker/Dockerfile.linux.arm64 [hunk 3]
```diff
@@ -5,7 +5,7 @@ LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
   org.label-schema.vendor="Drone.IO Community" \
   org.label-schema.schema-version="1.0"
 
-RUN apk add --no-cache bash git curl rsync openssh-client py3-pip py3-requests python3-dev libffi-dev libressl-dev libressl build-base && \
+RUN apk add --no-cache bash git curl rsync openssh-client py3-pip py3-requests py3-boto3@testing python3-dev libffi-dev libressl-dev libressl build-base && \
   pip3 install -U pip && \
   pip3 install ansible==2.8.4 && \
   apk del python3-dev libffi-dev libressl-dev build-base
```
Comment (tboerger) [inline docker/Dockerfile.linux.arm64]:
```patch
-  apk del python3-dev libffi-dev libressl-dev build-base
+  apk del --no-cache python3-dev libffi-dev libressl-dev build-base
```

**Comment by mtb-xt** [comment]
@tboerger ready to merge, thanks for your patience and guidance

**Comment by tboerger** [comment]
Thank you also for your patience :)

**Comment by tboerger** [comment]
I have also pushed a tag version `v2.0.0`, this should result in Docker tags `2.0.0`, `2.0` and `2` which includes python3 and the boto3 module.

**Pull request merged**